docs/data/data-grid/demo/demo.md

```diff
@@ -6,7 +6,7 @@ title: Data Grid - Demo
 
 <p class="description">Use the demo below to explore the available features.</p>
 
-{{"demo": "FullFeaturedDemo.js", "defaultCodeOpen": false, "bg": "inline"}}
```

Comment:
Should we delete the `FullFeaturedDemo` if it's not being used? Maybe it's used for Argos?

Reply:
I'm using it as the demo on the row virtualization line.

Reply:
@DanailH, I'd use another demo that is specific for row virtualization and delete the FullFeaturedDemo, but just remembered I used this one because there's not another option for row-virtualization.

Reply:
I am also in favor of removing `FullFeaturedDemo` and create one more fine-tuned for row virtualization
Can be done in a follow up